Cargo.toml

```diff
@@ -23,7 +23,13 @@ members = [
 ]
 
 [profile.release]
-debug = true
+# Emit only the line info tables, not full debug info, in release builds, to
+# substantially reduce the size of the debug info. Line info tables are enough
+# to symbolicate a backtrace, but not enough to use a debugger interactively.
```

Comment:
Nit, I’d say it makes it harder to use a debugger interactively rather than impossible. People use debuggers without any symbols at all. IDA Pro has a debugger module ;)

Reply:
(And, if this made release build debugging truly *impossible*, I’d probably be against landing it regardless of the size savings).

Reply:
Sure, adjusting. You can always step through the asm, that's true. :D